src/main/java/com/minecolonies/coremod/generation/defaults/DefaultResearchProvider.java

```diff
@@ -615,6 +622,74 @@ public DefaultResearchProvider(final DataGenerator generator)
           .addItemCost(Items.DIAMOND, 1)
           .addEffect(ModBuildings.mysticalSite.getBuildingBlock(), 1)
           .addToList(r);
+
+        // Primary Research #6
+        final Research remembrance = new Research(new ResourceLocation(Constants.MOD_ID, "civilian/remembrance"), CIVIL)
+                .setTranslatedName("Remembrance")
+                .setSortOrder(6)
+                .setIcon(ModBlocks.blockHutGraveyard.asItem())
+                .addEffect(ModBuildings.graveyard.getBuildingBlock(), 1)
+                .addBuildingRequirement(ModBuildings.TOWNHALL_ID, 2)
+                .addItemCost(Items.BONE, 8)
+                .setTranslatedSubtitle("Our fallen shall not be forgotten!")
+                .addToList(r);
+
+        new Research(new ResourceLocation(Constants.MOD_ID, "civilian/undertakeremergency"), CIVIL)
+                .setParentResearch(remembrance)
+                .setTranslatedName("Undertaker Emergency")
+                .setTranslatedSubtitle("Teach Undertaker the ability to run towards graves")
```

Comment:
This is more for a "funny joke" than an explanation, this should probably be moved to line 134 (in this file). Otherwise 👍

Reply:
I'm not sure the line number is correct - or I missunderstand something.

Line 134 of this file is:
 effects.add(new ResearchEffect(ModBuildings.graveyard.getBuildingBlock()).setTranslatedName("Unlocks Graveyard"));
--> unlocking the graveyard

This tech is related to line 121:
 effects.add(new ResearchEffect(UNDERTAKER_RUN).setTranslatedName("Undertaker unlocks run ability"));
--> a new tech that gatekeep the ability of the undertaker to run towards grave as per request of the playtester masmc05 on discord

![image](https://user-images.githubusercontent.com/6545998/114276431-d5b86b80-9a26-11eb-9aba-1b25b1bc8386.png)


Should the setTranslatedSubtitle still be move?

Reply:
hmm, when I was viewing the diff it said line 134 was for this research. In that case, line 121 😄

Reply:
I made the change you requested (not committed). However, it's inconsistent with the rest of the research definition and the tooltip is not displaying the line anymore

Updated version with your suggestion:

![image](https://user-images.githubusercontent.com/6545998/114278432-22547480-9a30-11eb-81b0-893daaa868f5.png)

Should I revert my change or do you prefer it to be set as you requested?

Reply:
How is it inconsistent? Not all researches have subtitles.

Reply:
Research effects can have separate subtitles from the research itself, if you wanted a more detailed description of what the effect itself does.

Reply:
In this case I was aiming to have a subtitle to the research. Subtitle to the research are set in the "new Research" itself and not on the new Effect. 

I don't know where the effect subtitle are displayed in the interface. Feel free to ping me on discord if it's easier to discuss.
If you really want it on the effect, I just need to push my pending changes but it makes more sense on the research for me.

Reply:
Update: grywo (probably gattsuru) let me know that my misunderstanding is coming from another bug that he is fixing rigth now. I will commit the change to move the text to the effect subtitle so we can continue on this PR. The text will appear in the UI when grywo will fix the bug related to it.